web/src/lib/domain-logic/allowFormation.ts

```diff
@@ -10,15 +10,7 @@ export const allowFormation = (
   legalStructureId: string | undefined,
   persona: BusinessPersona | undefined
 ): boolean => {
-  const featureFlagMap: Partial<Record<FormationLegalType, boolean>> = {
-    "foreign-limited-liability-company": process.env.FEATURE_BUSINESS_FLC === "true",
-    "foreign-limited-liability-partnership": process.env.FEATURE_BUSINESS_FLLP === "true",
-    "foreign-limited-partnership": process.env.FEATURE_BUSINESS_FLP === "true",
-    "foreign-s-corporation": process.env.FEATURE_BUSINESS_FCORP === "true",
-    "foreign-c-corporation": process.env.FEATURE_BUSINESS_FCORP === "true",
-    "foreign-nonprofit": process.env.FEATURE_BUSINESS_FNP === "true",
-    nonprofit: process.env.FEATURE_BUSINESS_NP === "true",
```

Comment:
[pebble/boulder] We can remove these env variables from `next.config.js`, `jest.config.ts`, and `.circleci/config.yml`. This is the only place in the app we're referencing these. If you do a global search for "FEATURE_BUSINESS_" and all of the instances should come up.

Not a code change per se, but we can also remove these from the .env template in bitwarden.

Reply:
This is now complete. However, I'm going to have to wait on @aloverso to provide permissions to edit the `.env` template in Bitwarden for now, if that's okay.